src/WebJobs.Script.WebHost/WebHost/ScriptWebHostEnvironment.cs

```diff
@@ -26,7 +26,7 @@ public ScriptWebHostEnvironment(IEnvironment environment)
             _delayTaskCompletionSource.SetResult(null);
         }
 
-        public bool DelayRequestsEnabled => DelayCompletionTask.IsCompleted;
+        public bool DelayRequestsEnabled => !DelayCompletionTask.IsCompleted;
```

Comment:
?!
Hope there's a test for this now....

Reply:
There's not -- I thought the Linux test was failing due to this (it returns a 404 when it should be waiting and return a 401). But that wasn't the cause.

Is my change correct? I was trying to work out in my head...

I'll see if I can add a test to make sure the middleware that uses this is actually working like we expect.

Reply:
My original tests added in https://github.com/Azure/azure-functions-host/pull/2777 tested the Assign blocking logic for Linux

Reply:
So something isn't working right here -- if we have a test, then it must not be checking the right thing. I'll validate it tomorrow.

Reply:
The test was disabled. My latest changes get this working again and I verified that flipping this back failed; with the negation, it passes.